src/NzbDrone.Core/Download/Pending/PendingReleaseReason.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NzbDrone.Core.Download.Pending
+{
+    public enum PendingReleaseReason
+    {
+        Delay = 0,
+        StoreAndForward = 1
```

Comment:
I don't think the PendingReleaseReason has any effect on what happens with it. Everything is re-evaluated on RssSync afaik.
`Delay` is caused by a DecisionSpec.
`StoreAndForward` is if sending to Downloader Failed or downloader Disabled.
`Fallback` is any release that didn't get rejected but wasn't the best available.

I don't think there's a difference between StoreAndForward (horrible name) and Fallback.

Delay & StoreAndForward items should be shown on the UI, dunno about Fallback, I think we should filter those.